websocket-autobahn/config/fuzzingclient.json

```diff
@@ -0,0 +1,12 @@
+{
+  "outdir": "./reports/servers",
+  "servers": [
+    {
+      "agent": "actix-web-actors",
+      "url": "ws://host.docker.internal:9001"
+    }
+  ],
+  "cases": ["*"],
+  "exclude-cases": [],
```

Comment:
Oh, I also meant to ask - are there any tests we should be excluding by default? Not sure if `actix` is already supporting/planning-on-supporting per-message deflate and whatever else

Reply:
Might as well for now. Looks like it's `"12.*"` and `"13.*"`.

Reply:
51201fa807600c3bb0f68c4f06c0dc08cd073f01